src/safe-upgrade-tool/introduction.md

```diff
@@ -0,0 +1,20 @@
+---
+group: software-update-guide
+title: Introduction
+ee_only: True
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool ALPHA (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed in it.
+
+The SUT identifies potential problems that must be fixed in your custom code before attempting to upgrade to a newer version of Magento. SUT complements another experimental tool called [ampersand-magento2-upgrade-patch-helper](https://github.com/AmpersandHQ/ampersand-magento2-upgrade-patch-helper).
```

Comment:
Please remove the mention to ampersand "SUT complements another experimental tool called [ampersand-magento2-upgrade-patch-helper](https://github.com/AmpersandHQ/ampersand-magento2-upgrade-patch-helper)."

Reply:
done